hack/run-e2e-conformance-virtual-ocp.sh

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env bash
+set -xeo pipefail
+
+cluster_name=${CLUSTER_NAME:-ocp-virt}
+domain_name=lab
+
+api_ip=${API_IP:-192.168.122.253}
```

Comment:
AFAIR, 192.168.122.0/24 a default libvirt network. will it work if this network isn't created? IMO, it's better to create a separate network